### PR TITLE
Handle the case when min pods is nil for "kubectl get hpa"

### PR DESCRIPTION
**Pull request by janetkuo**
Follow up #15902; ref #15957

### CODE REVIEW

File: docs/man/man1/kubectl-autoscale.1 [hunk 1]
```diff
@@ -72,7 +72,7 @@ An autoscaler can automatically increase or decrease number of pods deployed wit
     If non\-empty, sort list types using this field specification.  The field specification is expressed as a JSONPath expression (e.g. 'ObjectMeta.Name'). The field in the API resource specified by this JSONPath expression must be an integer or a string.
 
 .PP
-\fB\-t\fP, \fB\-\-template\fP=""
+\fB\-\-template\fP=""
```
Comment (JanetKuo) [inline docs/man/man1/kubectl-autoscale.1]: #15996; please ignore it

File: pkg/kubectl/resource_printer.go [hunk 2]
```diff
@@ -1396,7 +1396,10 @@ func printHorizontalPodAutoscaler(hpa *extensions.HorizontalPodAutoscaler, w io.
 	if hpa.Status.CurrentCPUUtilizationPercentage != nil {
 		current = fmt.Sprintf("%d%%", *hpa.Status.CurrentCPUUtilizationPercentage)
 	}
-	minPods := hpa.Spec.MinReplicas
+	minPods := "<unknown>"
```
Comment (lavalamp) [inline pkg/kubectl/resource_printer.go]: Perhaps "<unset>"? Presumably if it can be nil, it's optional?

Comment (JanetKuo) [inline pkg/kubectl/resource_printer.go]: cc @piosz @gmarek we should use the same value in describe and get.

Comment (piosz) [inline pkg/kubectl/resource_printer.go]: I think there is no difference what the default would be. It should never happen indeed.

Comment (JanetKuo) [inline pkg/kubectl/resource_printer.go]: Agree.

Comment (lavalamp) [inline pkg/kubectl/resource_printer.go]: sorry, github ate my text-- I meant "`<unset>`"

Comment (JanetKuo) [inline pkg/kubectl/resource_printer.go]: I like `"<unset>"`.

Comment (JanetKuo) [inline pkg/kubectl/resource_printer.go]: Changed to `"<unset>"`

**Comment by lavalamp** [comment]
LGTM

**Comment by lavalamp** [comment]
Thanks!

**Pull request merged**